tox.ini

```diff
@@ -0,0 +1,34 @@
+[tox]
+envlist = {py27,py32,py33,py34}-{1.6,1.7,1.8}.X,flake8,coverage,docs
+
+[testenv]
+basepython =
+	py27: python2.7
+  py32: python3.2
+	py33: python3.3
+	py34: python3.4
+deps =
+  1.6.X: Django>=1.6,<1.7
+	1.7.X: Django>=1.7,<1.8
+	1.8.X: Django>=1.8,<1.9
+  -rrequirements-tests.txt
+commands = {envpython} tests/runtests.py
+
+[testenv:flake8]
+basepython = python3.4
+commands = pylint --rcfile=.pylintrc -E dbbackup/ || true
+           flake8 dbbackup || true
+deps = -rrequirements-tests.txt
+
+[testenv:coverage]
+passenv = TRAVIS TRAVIS_JOB_ID TRAVIS_BRANCH
+basepython = python3.4
+commands = coverage run tests/runtests.py
+           coverage report -m --fail-under 50
+deps = -rrequirements-tests.txt
```

Comment:
You're right, it doesn't need one.  Putting it in a test env is mostly a matter of convenience.  This way when you run Tox locally on a branch it will give you back the coverage results as well without having to run coverage separately and it will still run coverage on Travis.